### ISSUE TITLE
Non-interactive env creation

### ISSUE THREAD

**Issue by danthegoodman1** [danthegoodman1/EpicEnv#11]
Defaults using flags

**Comment by linamez** [comment]
I am wondering what is the best way to use EpicEnv in a production environment, for which interactive env creation is not possible. Is there a non-interactive way to decrypt and export env variables from EpicEnv ?

Thanks !

**Comment by linamez** [comment]
Yes you are right, I meant sourcing one rather than creating it !

**Comment by danthegoodman1** [comment]
Sourcing is all non-interactive. Are you running into something?
